origin-dapp/src/utils/mobile.js

```diff
@@ -0,0 +1,24 @@
+const userAgent = navigator.userAgent || navigator.vendor || window.opera
+
+export default function isMobile() {
```

Comment:
I'm generally a proponent of using fewer vars and avoiding any chance at data corruption. (aka data normalization) Seems like `isMobile` and `mobileDevice` could be consolidated since a `null` value on mobileDevice means that we're not on mobile.

Having both opens up the possibility of `isMobile` being true and `mobileDevice` being null (or vice versa) which would lead to strange behavior. 

@micahalcorn You have opinions on this?

Reply:
@wanderingstan I agree, but I suspect that @Natasha08 was keeping the old detection method so that we maintained the ability to distinguish between mobile vendors. Should we keep both methods and rename `mobileDevice` to avoid confusion, or should we just return the OS vendor instead of `true` in a single detection method?

Reply:
Yeah, we should drop `isMobile()` and add it's logic as fallback in `mobileDevice()`  such as 
```
export function mobileDevice() {
  if (/android/i.test(userAgent)) {
    return 'Android'
  } else if (/iPad|iPhone|iPod/.test(userAgent)) {
    return 'iOS'
  } else if (userAgent.match(/Mobi/)) {
    return 'Unknown Mobile'
  } else {
    return null
  }
}

@Natasha08 what do you think?

```